File: FinalProjectGaming/Model/Genres.cs

public class Genres
{
    public int count { get; set; }
    public Result[] results { get; set; }
}

public class Result
{
    public int id { get; set; }
    public string name { get; set; }
    public string slug { get; set; }
    public int games_count { get; set; }
    public string image_background { get; set; }
    public Game[] games { get; set; }
}

public class Game
{
    public int id { get; set; }
    public string slug { get; set; }
    public string name { get; set; }
    public int added { get; set; }
}

File: FinalProjectGaming/ClientApp/src/app/genres.ts
export interface genres {
    count:    number;
    results:  Result[];
}

export interface Result {
    id:              number;
    name:            string;
    slug:            string;
    gamesCount:      number;
    imageBackground: string;
    games:           Game[];
}

export interface Game {
    id:    number;
    slug:  string;
    name:  string;
    added: number;
}
